MatrixSDK/LocationSharing/MXBeaconInfoSummary.swift

```diff
@@ -22,18 +22,18 @@ public class MXBeaconInfoSummary: NSObject, MXBeaconInfoSummaryProtocol {
     
     // MARK: - Properties
     
-    public let identifier: String
+    public let id: String
     public let userId: String
     public private(set) var deviceId: String?
     public private(set) var beaconInfo: MXBeaconInfo
     public private(set) var lastBeacon: MXBeacon?
         
-    public var expiringDate: UInt64 {
+    public var expiryTimestamp: UInt64 {
         return beaconInfo.timestamp + beaconInfo.timeout
     }
     
     public var hasExpired: Bool {
-        return Date().timeIntervalSince1970 > TimeInterval(expiringDate)
+        return Date().timeIntervalSince1970 * 1000 > TimeInterval(expiryTimestamp)
```

Comment:
Technically it should probably be written like so but it's clearly equivalent 🙃
```suggestion
        return Date().timeIntervalSince1970 > TimeInterval(expiryTimestamp / 1000)
```